src/Model/User.php

```diff
@@ -260,6 +260,7 @@ public function setSuperAdmin(bool $boolean): void
     public function setPlainPassword(?string $password): void
     {
         $this->plainPassword = $password;
+        $this->updatedAt = new \DateTime();
```

Comment:
```suggestion
        // Do not remove this, it will trigger preUpdate doctrine event
        // when you only change the password, since plainPassword 
        // is not persisted on the entity, doctrine does not watch for it.
        $this->updatedAt = new \DateTime();
```